packages/vite/src/node/preview.ts

```diff
@@ -48,7 +48,8 @@ export interface PreviewServer {
    */
   httpServer: http.Server
   /**
-   * The resolved urls Vite prints on the
+   * The resolved urls Vite prints on the CLI. null in middleware mode or
+   * before `server.listen` is called.
```

Comment:
```suggestion
   * The resolved urls Vite prints on the CLI
```

The second sentence can be removed as it only applies for the dev server.

Reply:
Thanks for catching this! I didn't notice the different signatures 😅